Jenkinsfile

```
@@ -102,54 +102,134 @@ pipeline {
+              def config_path = 'CONFIG_PATH=/usr/src/app/config';
+              def api_access_mode = '--env API_ACCESS_TYPE=internal ';
+
+              // SQLite Config
+              def db_storage_folder_path = "$WORKSPACE/process_engine_databases";
+              def db_storage_path_correlation = "process_engine__correlation_repository__storage=$db_storage_folder_path/correlation.sqlite";
+              def db_storage_path_external_task = "process_engine__external_task_repository__storage=$db_storage_folder_path/external_task.sqlite";
+              def db_storage_path_process_model = "process_engine__process_model_repository__storage=$db_storage_folder_path/process_model.sqlite";
+              def db_storage_path_flow_node_instance = "process_engine__flow_node_instance_repository__storage=$db_storage_folder_path/flow_node_instance.sqlite";
+
+              def db_environment_settings = "jenkinsDbStoragePath=${db_storage_folder_path} ${db_storage_path_correlation} ${db_storage_path_external_task} ${db_storage_path_process_model} ${db_storage_path_flow_node_instance}";
+
+              def npm_test_command = "node ./node_modules/.bin/cross-env NODE_ENV=test-sqlite JUNIT_REPORT_PATH=process_engine_runtime_integration_tests.xml CONFIG_PATH=config API_ACCESS_TYPE=internal ${db_environment_settings} mocha -t 200000 test/**/*.js test/**/**/*.js";
+
+              docker.image("node:${NODE_VERSION_NUMBER}").inside("--env PATH=$PATH:/$WORKSPACE/node_modules/.bin") {
+                sqlite_exit_code = sh(script: "${npm_test_command} --colors --reporter mocha-jenkins-reporter --exit > process_engine_runtime_integration_tests.txt", returnStatus: true);
+
+                sqlite_testresults = sh(script: "cat process_engine_runtime_integration_tests.txt", returnStdout: true).trim();
+                junit 'process_engine_runtime_integration_tests.xml'
+              };
+
+              sh('cat process_engine_runtime_integration_tests.txt');
+
+              sqlite_tests_failed = sqlite_exit_code > 0;
+            }
+          }
+        }
+        stage('PostgreSQL') {
+          agent any
+          options {
+            skipDefaultCheckout()
+          }
+          steps {
+            unstash('post_build');
+
+            script {
+              // Postgres Config
+              def postgres_host = "postgres";
+              def postgres_username = "admin";
+              def postgres_password = "admin";
+              def postgres_database = "processengine";
+
+              def postgres_settings = "--env POSTGRES_USER=${postgres_username} --env POSTGRES_PASSWORD=${postgres_password} --env POSTGRES_DB=${postgres_database}";
+
+              def db_storage_path_correlation = "process_engine__correlation_repository__host=${postgres_host} process_engine__correlation_repository__username=${postgres_username} process_engine__correlation_repository__password=${postgres_password} process_engine__correlation_repository__database=${postgres_database}";
```

Overriding the hostname should suffice here.
All other parameters are equivalent to the default config.
```suggestion
              def db_database_host_correlation = "process_engine__correlation_repository__host=${postgres_host}";
```